Resources/Prototypes/_Goobstation/Alerts/blob.yml

```
@@ -0,0 +1,169 @@
+  - sprite: /Textures/_Goobstation/Blob/Interface/Alerts/blob_health.rsi
+    state: point17
+  - sprite: /Textures/_Goobstation/Blob/Interface/Alerts/blob_health.rsi
+    state: point18
+  - sprite: /Textures/_Goobstation/Blob/Interface/Alerts/blob_health.rsi
+    state: point19
+  - sprite: /Textures/_Goobstation/Blob/Interface/Alerts/blob_health.rsi
+    state: point20
+  name: blob-health-alert-name
+  description: blob-health-alert-desc
+  minSeverity: 0
+  maxSeverity: 20
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)